pages/api/examples/sign_document.ts

```diff
@@ -1,70 +1,96 @@
 
-import fs from 'fs'
-import {plainAddPlaceholder}  from 'node-signpdf'
-import nodesignpdf  from 'node-signpdf'
+import fs from "fs";
+import { plainAddPlaceholder } from "node-signpdf";
+import nodesignpdf from "node-signpdf";
 
-import { Web3Storage, File } from 'web3.storage'
+import { Web3Storage, File } from "web3.storage";
 
-import {IncomingForm} from 'formidable';
+import { IncomingForm } from "formidable";
+
+const { PDFDocument } = require("pdf-lib");
 
 export const config = {
   api: {
-    bodyParser: false
-  }
+    bodyParser: false,
+  },
 };
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
-  const session = await auth(req, res)
+  const session = await auth(req, res);
 
   if (session) {
-
-    const data:any = await new Promise((resolve, reject) => {
+    const data: any = await new Promise((resolve, reject) => {
       const form = new IncomingForm();
       form.parse(req, (err, fields, files) => {
-          if (err) return reject(err);
-          resolve({ fields, files });
+        if (err) return reject(err);
+        resolve({ fields, files });
       });
     });
-    let pdfBuffer = fs.readFileSync(data.files.file[0].filepath)
-
-        pdfBuffer = plainAddPlaceholder({
-          pdfBuffer,
-          reason: 'I have reviewed it.',
-          signatureLength: 1612,
-        })
-      
-        console.log('add plainAddPlaceholder')
-        const signedPdf = nodesignpdf.sign(
-            pdfBuffer,
-            fs.readFileSync('certificate.p12'),
-            {passphrase:"1234"}
-          );
-        
-        console.log('after sign')
-        const token = process.env.WEB3_STORAGE_TOKEN || "<NOT SET>"
-      
-        console.log('after web3.storage token')
-        const storage = new Web3Storage({ token })
-        console.log('after web3.storage instantiated')
-        const cid = await storage.put([new File([signedPdf], "signedPdf")])
-
-        console.log('cid:'+cid)
-        return res.send({
-          content:
-            `CID:${cid} This is protected content. You can access this content because you are signed in.`
-        })      
 
+    let pdfBuffer = fs.readFileSync(data.files.file[0].filepath);
+
+    pdfBuffer = plainAddPlaceholder({
+      pdfBuffer,
+      reason: "I have reviewed it.",
+      signatureLength: 1612,
+    });
+
+    console.log("add plainAddPlaceholder");
+    const signedPdf = nodesignpdf.sign(
+      pdfBuffer,
+      fs.readFileSync("certificate.p12"),
+      { passphrase: "1234" }
+    );
+
+    // Create a new PDFDocument from the signed PDF
+    const pdfDoc = await PDFDocument.load(signedPdf);
+
+    // Continue with adding the image
+    const page = pdfDoc.getPage(0); // Get the first page or the specific page you want to add the image to
+    const imageBytes = fs.readFileSync(
+      "/Users/woonkoh/Desktop/tublian/SignDocumentNext/pages/api/examples/signature.png"
```

Comment:
change this so that it reads a image file from the local relative path, and just through an example signature.png into the repository for it to read.

Reply:
I changed it to relative path, and added the example signature.png file into the repository.